lib/rubocop/cop/rspec/mixin/top_level_group.rb

```diff
@@ -29,6 +34,7 @@ def on_top_level_example_group(_node); end
         def on_top_level_group(_node); end
 
         def top_level_group?(node)
+          warn DEPRECATED_PRIVATE_METHOD_WARNING
           top_level_groups.include?(node)
         end
 
```

Comment:
Something feels off about the `node.children` line.
I vaguely recall that a single-statement block and a multi-statement block’s AST are different (yuck!). Will the result be correct in both cases?
Like
`foo { RSpec.describe { } }` vs `foo { RSpec.describe { }; RSpec.shared_examples { } }`?


Unrelated to this PR, just an observation.
Might as well be a false alarm, and I apologise in advance.

Reply:
Sounds like more specs are needed!